src/components/Beer.vue

```diff
@@ -1,22 +1,24 @@
 <template>
   <div class="beer mt-5" @click="editMode = !editMode">
+    <img :src="beer.image" class="label"/>
     <keg :percent="percent" ></keg>
-    <div>
-      <img :src="beer.image" class="label"/>
-      <h1 class="display-2 text-truncate">{{ beer.beer_name }}</h1>
-      <h3 class="display-1 text-no-wrap font-weight-light">{{ beer.brewery.brewery_name }}</h3>
-    </div>
-    <br/>
-    <div class="description">
-      <h5 class="headline">
-        <v-layout>
-          <v-flex xs8>{{ beer.beer_style }}</v-flex>
-          <v-flex xs2>{{ beer.beer_abv }}% ABV</v-flex>
-          <v-flex xs2>{{ beer.beer_ibu }} IBU</v-flex>
-        </v-layout>
-      </h5>
-      <p class="subheading condensed">{{ beer.beer_description || 'No description'}}</p>
-      <h5 class="subheading font-weight-light">Tapped {{ beer.tapped | formatDate }}</h5>
+    <div class="content-wrapper">
+      <div class="heading-wrapper">
+        <h5 class="subheading font-weight-light"><span>{{ beer.brewery.brewery_name }} - </span>Tapped {{ beer.tapped | formatDate }}</h5>
```

Comment:
are we using snake_case here because this beer data is coming from the untapped api or?

Reply:
Yes